src/Teapot.Web/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Would it be worthwhile putting in some proper logging with app insights? Right now we don't really do much, so it's really hard to work out what's going wrong and resulting in the experience of #89

Reply:
Yep, it's easy enough, I'll add that in.